src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -7,31 +7,27 @@ public class FigureSupplier {
     private static final int MAX_FIGURE_NUMBER = 5;
     private static final int MAX_DEFAULT_VALUE = 10;
     private static final String DEFAULT_COLOR = "white";
-    private Random random;
-    private ColorSupplier colorSupplier;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier(random);
```

Comment:
Let's make these fields final and pass the instances to the constructor as in the previous hw